tagstudio/src/qt/ts_qt.py

```diff
@@ -416,6 +431,20 @@ def start(self):
         macros_menu.addAction(self.sort_fields_action)
 
         folders_to_tags_action = QAction("Create Tags From Folders", menu_bar)
+        show_libs_list_action = QAction("Show Libraries List", menu_bar)
```

Comment:
Similar suggestion to the main "Recent Libraries" one, just changing the menu action to match.
```suggestion
        show_libs_list_action = QAction("Show Recent Libraries", menu_bar)
```